### PR TITLE
Fix LaTeX code color in pir2.forumeiros.com and app.estuda.com

### PR DESCRIPTION
**Pull request by DasGestirn**
The LaTeX images are being displayed in black color, and considering the dark background, it's very difficult to understand them. The proposed change aims to invert the color of these images to white, as to make them easy to read.

Examples of the change on these sites can be seen in:

https://pir2.forumeiros.com/t205435-probabilidade-banca-fgv#705383

https://app.estuda.com/questoes/?id=11473523

### CODE REVIEW

**Comment by DasGestirn** [comment]
pir2forumeiros.com post with LaTeX code before the fix:

image

After:
image

---

app.estuda.com before the fix:

image

After:
image

**Comment by DasGestirn** [comment]
After spending a few more time thinking, wouldn't it be better to use this solution for inverting the LaTeX images anywhere else? 

It could be done by adding the `INVERT
img[src*="latex"]` in the `*` section of the `inversion-fixes.config` file.

Is it a sound idea? I haven't yet seen a site where this solution wouldn't work well.

If it's a reasonable solution, i will close this pull request and another with the proposed change.

File: src/config/dynamic-theme-fixes.config [hunk 1]
```diff
@@ -30885,3 +30885,17 @@ CSS
...
+================================
+
+pir2.forumeiros.com
+
+INVERT
+img[src*="latex"]
+
+================================
+
+app.estuda.com
```
Comment (alexanderby) [inline src/config/dynamic-theme-fixes.config:30898]: Please preserve the alphabetical order.

**Review by alexanderby — CHANGES_REQUESTED** [review]

File: src/config/dynamic-theme-fixes.config (hunk 1 above)
Comment (DasGestirn) [reply src/config/dynamic-theme-fixes.config]: Hi Alexander!

Sorry for not inserting my fixes in alphabetical order, i should have read the documentation first.

**Comment by DasGestirn** [comment]
Hi @alexanderby !

I know you're probably busy with more pressing matters at the moment, but it won't take much time (5 minutes at most) to review this pull request and merge it.

I've already corrected the alphabetical order of the fixes i inserted and i also checked it doesn't break anything. It's a pretty simple correction.

**Review by alexanderby — APPROVED** [review]

**Comment by alexanderby** [comment]
All merged, thank you.

**Pull request merged**